src/login/logind.conf.in

```diff
@@ -42,7 +42,7 @@
 #IdleAction=ignore
 #IdleActionSec=30min
 #RuntimeDirectorySize=10%
-#RuntimeDirectoryInodesMax=400k
+#RuntimeDirectoryInodesMax=409600
```

Comment:
The header of this file says the value is compile-time default. But `RuntimeDirectoryInodesMax=` is calculated from `RuntimeDirectorySize=` by default... So,
1. keep this as is,
2. make this blank,
3. add a brief comment,
4. update the header.

Which options do you like?